methods/main_methods.py

```diff
@@ -13,29 +13,29 @@
 HEIGHT_INDEX = config.height_index
 
 ## Boundary Detection
-def sobel(x):
```

Comment:
Most of the changes in main_methods change variable names: x -> data, X -> data, O -> outliers, and L -> labels